data_management/opensearch_indexer/opensearch_indexer/index_file_content_and_metadata_in_opensearch.py

```diff
@@ -87,6 +87,11 @@ def _fetch_file_data(
     session.close()
 
     file_data = {
+        **{
```

Comment:
I think it is worth us documenting this in a new ADR as it changes the initial decision in ADR 17

Reply:
@anthonyhashemi added a "Modification" section at the bottom of the document explaining the change

Reply:
cheers, I will take a look